control-plane/api/v1alpha1/proxydefaults_types_test.go

```diff
@@ -5,12 +5,13 @@ import (
 	"testing"
 	"time"
 
-	"github.com/hashicorp/consul-k8s/control-plane/api/common"
 	capi "github.com/hashicorp/consul/api"
 	"github.com/stretchr/testify/require"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/validation/field"
+
```

Comment:
I suspect this is an IDE formatting thing but I suspect we want this to be reverted.